src/shared/extensionUtilities.ts

```diff
@@ -336,21 +359,59 @@ export async function aboutToolkit(): Promise<void> {
  * Returns a string that includes the OS, AWS Toolkit,
  * and VS Code versions.
  */
-export function getToolkitEnvironmentDetails(): string {
+export async function getToolkitEnvironmentDetails(): Promise<string> {
```

Comment:
why async?

Reply:
Artifact from a past iteration I need to revert